components/RenderProposalContentLambda.tsx

```diff
@@ -315,9 +354,22 @@ export const contentToData = (
   return data;
 };
 
-const RenderProposalContentLambda = ({ data }: { data: data }) => {
-  const [hasParam, setHasParam] = useState(false);
-
+const RenderProposalContentLambda = ({
+  data,
+  isOpenToken: isOpenToken = false,
+}: {
+  data: data;
+  isOpenToken?: boolean;
+}) => {
+  let defaultOpen = false;
+  if (
+    isOpenToken &&
+    (data.type == "TransferFA1_2" ||
+      data.type == "ApproveFA1_2" ||
+      data.type == "TransferFA2")
+  )
+    defaultOpen = true;
+  const [hasParam, setHasParam] = useState(defaultOpen);
```

Comment:
this code need only to be run 1 time:
```suggestion
  const [hasParam, setHasParam] = useState(() =>  isOpenToken &&
    (data.type == "TransferFA1_2" ||
      data.type == "ApproveFA1_2" ||
      data.type == "TransferFA2"));
```